### DISCUSSION TITLE
How can I delete a Supabase organization associated with a deleted Vercel team?

### DISCUSSION THREAD

**Discussion by taykim01** [supabase/supabase#30343] [Questions]
<img width="1002" alt="Screenshot 2024-11-07 at 09 59 01" src="https://github.com/user-attachments/assets/364114a8-49c4-4a28-afe3-cff6f2d7bc6d">

This is all I am seeing. Does anybody know how to delete this Supabase organization/project?

**Comment by NTTrinh2003** [comment]
Same problem here :((( Did you find solution yet?

**Reply by NTTrinh2003** [reply]
- First, select the team that you want to remove integration (top left with arrows)  
- Second, select the database tab, then choose supabase  
- Next, select Settings -> Delete database -> Confirm  
- Then check back to integration (Supabase database was uninstalled) -> Settings -> Uninstall integration  
- Go back to Supabase -> Organization deleted  
Hope this will help anyone encounter the same problem :)))

**Comment by fourcolors** [comment]
Yeah I'm basically never installing through Vercel again. Such a bad idea. Why would anyone ever want to make things harder?
